ui/raidboss/data/02-arr/trial/ifrit_nm.js

```diff
@@ -42,8 +42,8 @@
         'Surrender': 'Surrender', // FIXME
       },
       'replaceText': {
-        '(inner)': '(innen)',
-        '(outer)': '(außen)',
+        'inner': 'innen',
```

Comment:
For your own knowledge, the reason this caused double parentheses is that the `(inner)` is treated as a regular expression.  I think it would need to be `'\\(inner\\)': '(innen)'` to work properly as it was intended to before.  I think this is fine, because there is no other "inner" or "outer" things to replace, however I think other files might have this same issue.

Reply:
In order to understand us well:
- do you want me to put '\\ (inner \\)': '(innen)'?
- or do I leave as I put?

Reply:
I think either is fine here.  I was mentioning it as information for other timelines.  Sometimes not having the parentheses can cause conflicts where two translations apply to the same log.  The `npm run test` step will definitely tell you when this is the case.  So if the test passes, it is not needed.

Reply:
Very good